bot/src/plugins/github_issues.cr

```diff
@@ -1,8 +1,19 @@
 require "framework/plugin"
+require "http/client"
+require "json"
 
 class GithubIssues
   include Framework::Plugin
 
+  def get_issue msg, issue_id
+    r = HTTP::Client.get("https://api.github.com/repos/#{config.repositories[msg.channel.name]}/issues/#{issue_id}", headers: HTTP::Headers{"Accept" => ["application/vnd.github.v3+json"], "User-Agent" => ["cbot_issue_fetch"]})
```

Comment:
Please don't use single letter variable names, it's just not worth it. Also this line could use some line breaks ;)